packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -643,4 +641,46 @@ describe('SpinButton', () => {
 
     expect(onIncrement).toBeCalled();
   });
+
+  it('should fire custom onDecrement handler (with minimal properties)', () => {
+    const onDecrement: jest.Mock = jest.fn();
+
+    const renderedDOM: HTMLElement = renderIntoDocument(
+      <SpinButton
+        label='label'
+        onDecrement={ onDecrement }
+      />
+    );
+
+    const buttonDOM: Element = renderedDOM.getElementsByClassName('ms-DownButton')[0];
+
+    ReactTestUtils.Simulate.mouseDown(buttonDOM,
+      {
+        type: 'mousedown',
+        clientX: 0,
+        clientY: 0
+      }
+    );
+
+    expect(onDecrement).toBeCalled();
```

Comment:
Consider testing that the callback is only invoked _once_ per action using [toHaveBeenCalledTimes](https://jest-bot.github.io/jest/docs/expect.html#tohavebeencalledtimesnumber). Can also be applied to the `onValidate` test.

Regardless of if you choose to swap assertions, thank you for adding test coverage.